transplant_guide.md

```diff
@@ -104,13 +105,16 @@ Neighborhoods _Elia_
 ![seattle neighborhoods](seattle-neighborhood-map.jpg "Seattle Neighborhoods")  
 
 **Pioneer Square**
-Very eclectic neighborhood. Lots of homeless. The oldest neighborhood in Seattle!
+The oldest neighborhood in Seattle! There are sometimes cheap studios and one bedrooms here, but more are getting renovated into expensive loft-like spaces.
+This is close to the football and baseball stadiums, so sometimes traffic can be bad, and if you have a car and no dedicated parking that can be a big problem. Several of the homeless shelters/missions are in this area, so there are quite a lot of homeless people in the area, which could be nerve-wracking walking home alone at night if you are not moving to Seattle from a city. It is just south of downtown, so walking to class is definitely doable.
```

Comment:
But FWIW there's no need to be afraid of homeless people. They are also people, and in my experience quite friendly.